modules/ROOT/pages/configure-acm.adoc

```diff
@@ -398,16 +396,11 @@ image::access-mapping.png[Access Mapping for CommunityApi]
 . Select the *Public can access the community* check box.
 . Navigate to the guest user profile link under the *Guest user profile* section, named *[Community Name] Profile*.
 . Select *Edit*.
-. In the *Standard Object Permissions* section:
-.. Enable *Read* permissions for:
-... *Documents*
+. In the *Standard Object Permissions* section, enable *Read* permissions for: *Documents*
```

Comment:
ditto re. colon; also end all these sentences with periods.
GLOBAL
I will stop marking.

Reply:
Thank you for noting it! I will fix these.